doc-build/action.yml

```
@@ -37,6 +37,11 @@ inputs:
       default: true
       required: false
       type: boolean
+  skip-json-build:
+      description: 'Whether to build json files or not'
```

```suggestion
      description: 'Whether to build json files or not.'
```
Missing dot. BTW, ``checkout`` description is also missing the dot.